CONTRIBUTING.md

```diff
@@ -18,30 +18,34 @@
 <summary><b>theme example</b></summary>
 
 To create a custom theme you need to define a colorscheme for each of vim's modes. Each mode has a `fg` and `bg` field for every lualine section.
+You can add special effects with `gui`. 
 This is really easy in lua. Here is and example of a gruvbox theme.
 
 ```lua
 local gruvbox = {  }
 
 local colors = {
-  black = "#282828",
-  white = '#ebdbb2',
-  red = '#fb4934',
-  green = '#b8bb26',
-  blue = '#83a598',
-  yellow = '#fe8019',
-
-  gray = '#a89984',
-  darkgray = '#3c3836',
-
-  lightgray = '#504945',
-  inactivegray = '#7c6f64',
+ -- color format { termguicolor, notermguicolor}
```

Comment:
as you said, cterm colors work really well now, we can remove them from here and let them be calculated by your function

Reply:
Maybe leave them as they are . Allowing theme authors to provide them will make it easier for them to fix if the algorithm choses a wrong color.
Also it's better if we donn't have to calculate it in runtime. Plus genarating those values from function is pretty easy . I've added cterm values to those last 4 themes in a few minutes .
We might just add instruction to do that when docs are written.

Reply:
I think you are right.